src/common/util/random.js

```diff
@@ -62,6 +62,13 @@ define(['chance', 'common/Constants'], function (ChanceJs, CONSTANTS) {
         return relid;
     }
 
+    function generateRandomString(length) {
+        return chance.string({
+            pool: relidPool,
+            length: length
+        });
+    }
+
```

Comment:
I added this method but ended up not using it.
I left it cause it can be handy (the relidPool) in other use cases.